migrations/versions/201607291704_da259a74e28_add_valid_login_num_to_user.py

```diff
@@ -0,0 +1,24 @@
+"""Add valid_login_num to User
+
+Revision ID: da259a74e28
+Revises: 23ef6a49ae28
+Create Date: 2016-07-29 17:04:53.295174
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'da259a74e28'
+down_revision = '23ef6a49ae28'
```

Comment:
There's a new revision in master so this should now be based on `28b674f8290d` (also in the comment in line 4). To keep the order in the file list up to date also bump the date in the filename so it comes after its parent revision.